### PR TITLE
Crm457 1122 Adds Prior authority test scenarios 

### PR DESCRIPTION
**Pull request by imtiazAhmd**
Prior authority end to end test scenario 1,2 and 7 from [Manual test script](https://docs.google.com/spreadsheets/d/1i97rgTQsUfZflPTJE_PPHfZizo8yW-u7ZGZfBvdHwt4/edit#gid=534567382)

https://dsdmoj.atlassian.net/browse/CRM457-1122

### CODE REVIEW

File: e2e/crm4/provider/not-prison-matter.spec.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import { test, expect } from '@playwright/test';
```
Comment (patrick-laa) [inline e2e/crm4/provider/not-prison-matter.spec.js:1]: Given this includes both prison law and not prison law scenarios, and the common theme is that the total value is too low, I think maybe there's a better file name than "not-prison-matter"

**Review by patrick-laa — COMMENTED** [review]
Looks good so far. Will the caseworker granting side of scenario 7 be added to this PR or be done in a separate one?

File: e2e/crm4/provider/not-prison-matter.spec.js (hunk 1 above)
Comment (imtiazAhmd) [reply e2e/crm4/provider/not-prison-matter.spec.js:1]: On second thought, I rather include caseworker side here. making this PR to draft again till then.

File: build_test_local.sh [hunk 2]
```diff
@@ -7,11 +7,14 @@ if [[ $(uname -m) == 'arm64' ]];
 then
   echo "Apple Silicon detected"
   export DOCKER_BUILDKIT=1
+  export LOCAL_PLATFORM=linux/amd64
```
Comment (jogilder) [inline build_test_local.sh:10]: is this something you spoke about with @jsugarman ? I wondered how it fitted in with his comments about this line not working for him?

Comment (imtiazAhmd) [reply build_test_local.sh:10]: had a session with @jsugarman and we found this works as intended.

File: e2e/crm4/provider/submit-an-application.spec.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+			await reasonWhyPage.fillReasonWhyForm();
+			// Expectations
+			await expect(page.getByRole('heading', { name: 'Your application progress' })).toBeVisible();
+			// Actions
+			await page.getByRole('link', { name: 'Submit application' }).click();
+			// Expectations
+			await expect(page.getByRole('heading', { name: 'Check your answers' })).toBeVisible();
+		});
+
+		await test.step('Check your answers', async () => {
```
Comment (patrick-laa) [inline e2e/crm4/provider/submit-an-application.spec.js:147]: Maybe this test could be called 'Check your answers and submit'?

**Review by patrick-laa — APPROVED** [review]

File: e2e/crm4/provider/submit-an-application.spec.js (hunk 3 above)
Comment (imtiazAhmd) [reply e2e/crm4/provider/submit-an-application.spec.js:147]: Creating another PR for restructuring. will rename there.

**Pull request merged**